### PR TITLE
2018-05-14-coding-period-begins.md: Add Post #1

### PR DESCRIPTION
**Pull request by ksdme**
Dated 14 May 2018

### CODE REVIEW

File: _posts/2018-05-14-coding-period-begins.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+I should let you know about the developments during the period and
+my plan ahead. Bonding Period is the time meant to help the students
+get started with the community, coding practices and discuss the details
+of the project one plans to follow with the community and their mentors.
+I had a great time knowing the community and the interesting stuff they do.
+
+<br/><br/>
+> ##### Developments
+
+1. **Resolved pending PR's**:
```
Comment (gaocegege) [inline _posts/2018-05-14-coding-period-begins.md]: Maybe we could list these pending PRs

File: _posts/2018-05-14-coding-period-begins.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  resolved and merged.
+
+2. <br/>**Finish newcomer process**: 
+  Since the deadline was too close when I joined the community 
+  I was instrcuted to work on my proposal first and worry with 
+  the newcomer process later on. I have since completed the 
+  newcomer process.<br/>
+
+3. <br/>**Schedule**: 
+  I worked with my primary(@gaocegege) and we have decided on a week
```
Comment (gaocegege) [inline _posts/2018-05-14-coding-period-begins.md]: I think the template does not support GitHub flavoured markdown. Then we should use `[@gaocegege](https://github.com/gaocegege)`

**Review by gaocegege — COMMENTED** [review]
Cool! LGTM :tada: 

Some nits.

File: _posts/2018-05-14-coding-period-begins.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  newcomer process.<br/><br/>
+
+3. **Schedule**: 
+  I worked with my primary(@gaocegege) and we have decided on a week 
+  on week schedule that I should follow to stay on track.<br/><br/>
+
+4. **Work flow**: 
+  We have discussed about the work flow I need to maintain and things
+  such as tooling etc.
+
```
Comment (gaocegege) [inline _posts/2018-05-14-coding-period-begins.md]: And we also update the details in projects.coala.io. It also takes us some time. I think we could add it here

File: _posts/2018-05-14-coding-period-begins.md (hunk 1 above)
Comment (ksdme) [reply _posts/2018-05-14-coding-period-begins.md]: Done.

File: _posts/2018-05-14-coding-period-begins.md (hunk 3 above)
Comment (ksdme) [reply _posts/2018-05-14-coding-period-begins.md]: > It also takes us some time.

What do you mean?

Comment (gaocegege) [reply _posts/2018-05-14-coding-period-begins.md]: I mean we need to list all you have done in the community bonding process. Then we need to list the update in projects.coala.io.

I am not sure if I express correctly. Do you get me? :thinking:

Comment (ksdme) [reply _posts/2018-05-14-coding-period-begins.md]: Yes, you mean that we need to post a reference to this list in projects.coala.io?

Comment (ksdme) [reply _posts/2018-05-14-coding-period-begins.md]: I have also updated this with a reference to projects.coala.io.

Comment (gaocegege) [reply _posts/2018-05-14-coding-period-begins.md]: Yeah. For example.

```markdown
5. Project description update

...
```

**Comment by ksdme** [comment]
@gaocegege, do you think this is good enough?

**Comment by gaocegege** [comment]
LGTM! :tada:

**Comment by ksdme** [comment]
Okay, So I am rebasing.

**Pull request merged**